homepairs/homepairsUI/tests/utilities/SyntaxValidator.test.tsx

```diff
@@ -159,4 +160,36 @@ describe('Test isEmptyOrSpaces Function', () =>{
         expect(isEmptyOrSpaces(value)).toBeFalsy();
     });
 
+});
+
+
+
+/** isPositiveWholeNumber Test * */
+const positiveWholeNumbers: string[] = [
+    '1',
+    '12345678910',
+    '90',
+    '10000',
+    '00000000',
+];
+
+const nonPositiveWholeNumbers: string[] = [
```

Comment:
it'd be nice to test an edge case closer than -1.0 for instance, see where -0.99 or -0.5 or -0.49 or -.001 go